### ISSUE TITLE
6.1.3 Why did I get worse when I increased the sample size?

### ISSUE THREAD

**Issue by fangxuehouwuming** [fchollet/deep-learning-with-python-notebooks#210]
Following the example in the book, I increased the sample size from 200 to 3000 and added a dropout regularization, but the final validation accuracy is only 50%.
![1661074009976](https://user-images.githubusercontent.com/74046524/185784611-5bb7c690-60f1-4530-a2d6-524572cfa46e.png)

**Comment by ifond** [comment]
YES!!! I have received your E-mail——Steven Lee

**Comment by lxl0506** [comment]
您好，攀枝花市文广旅局人事科已收到你的邮件！

**Comment by fangxuehouwuming** [comment]
My setting is training samples : validation samples =3000 : 10000 . Is it reasonable?
May be I know.

**Comment by Shuyib** [comment]
Maybe try to make a train,validation and test set. And see what happens?
